src/Misc/Part.h

```diff
@@ -19,6 +19,7 @@
 #include "../globals.h"
 #include "../Params/Controller.h"
 #include "../Containers/NotePool.h"
+#include "Sync.h"
```

Comment:
For just adding a pointer, an include is not needed. If you remove it and get warnings in Part.cpp, you can just add the include there.